requirements/redis-master.conf

```diff
@@ -0,0 +1,938 @@
+# Redis configuration file example
```

Comment:
Please move the sentinel `*.conf` files into a separate directory (maybe under `tests/redis-confs`)